src/Facebook/InstantArticles/Transformer/Rules/TimeRule.php

```diff
@@ -62,7 +62,7 @@ public function apply($transformer, $header, $node)
         $time_string = $this->getProperty(self::PROPERTY_TIME, $node);
         if ($time_string) {
             $time = Time::create($this->type);
-            $time->withDatetime(new \DateTime($time_string));
+            $time->withDatetime(new \DateTime($time_string, new \DateTimeZone('America/Los_Angeles')));
```

Comment:
Should we allow to set the default timezone as parameter in the rule transformation?